crates/bevy_ui/src/ui_node.rs

```diff
@@ -506,12 +509,19 @@ pub struct Style {
     /// <https://developer.mozilla.org/en-US/docs/Web/CSS/flex-basis>
     pub flex_basis: Val,
 
-    /// The size of the gutters between items in flexbox layout or rows/columns in a grid layout
+    /// The size of the gutters between items in a vertical flexbox layout or between rows in a grid layout
+    ///
+    /// Note: Values of `Val::Auto` are not valid and are treated as zero.
+    ///
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/row_gap>
+    pub row_gap: Val,
```

Comment:
```suggestion
    pub gap_width: Val,
```

Might be a little more intuitive and consistent? Not too bothered though.

Reply:
I personally would also prefer such a naming scheme. But it should be `gap_height` here, right?

Reply:
Hmm... well `row-gap` and `column-gap` are the standard CSS names. I feel like I'd prefer to stick to those if possible. Even though I also don't find this naming intuitive.